README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://github.com/pymc-devs/pymc3/blob/master/docs/pymc3_logo.jpg?raw=true
+.. image:: https://cloud.githubusercontent.com/assets/781659/26266864/0d462598-3c9c-11e7-8697-2e811e231694.png
```

Comment:
There might be references on out jpg logo. Deleting it can cause rendering problems. What about keeping both jpg and png logo in directory?

BTW, it's better to use direct link here